Adyen/client.py

```diff
@@ -121,6 +122,27 @@ def _determine_hpp_url(self, platform, action):
         result = '/'.join([base_uri, service])
         return result
 
+    def _determine_checkout_url(self, platform, service, action):
+        """This returns the Adyen API endpoint based on the provided platform,
+        service and action.
+
+        Args:
+            platform (str): Adyen platform, ie 'live' or 'test'.
+            service (str): API service to place request through.
+            action (str): the API action to perform.
+        """
+        if action == "paymentDetails":
+            action = "payment/details"
+        if action == "paymentResult":
+            action = "payment/result"
+        if action == "originKey":
+            action = "v1/originKeys"
+
+        base_uri = settings.BASE_CHECKOUT_URL.format(platform)
+        api_version = settings.CHECKOUT_API_VERSION
```

Comment:
for checkoutUtility (originKeys) the versioning is different(v1 only at the moment),
I would put api_version at the beginning and replace it in case the action is originKeys

Reply:
It is fixed now